app/controllers/AdviceStartController.scala

```diff
@@ -38,7 +38,8 @@ class AdviceStartController @Inject() (
   view: AdviceStartView,
   navigator: Navigator
 ) extends FrontendBaseController
-    with I18nSupport {
+    with I18nSupport
```

Comment:
Since every controller extends FrontendBaseController with I18nSupport with BaseController, can we just make the BaseController extends FrontendBaseController with I18nSupport, and then everyone of our controllers just extends BaseController?

Reply:
I did consider that, but favoured the linked trait in the ticket. Since you're saying this now, I'll try to do what you suggested :)